extensions/amp-story-shopping/0.1/amp-story-shopping-config.js

```diff
@@ -51,7 +51,9 @@ function keyByProductTagId(config) {
 export function storeShoppingConfig(pageElement, config) {
   const win = pageElement.ownerDocument.defaultView;
   return Services.storyStoreServiceForOrNull(win).then((storeService) => {
-    storeService?.dispatch(Action.ADD_SHOPPING_DATA, config);
+    const shoppingConfig = {};
+    shoppingConfig[pageElement.id] = config;
```

Comment:
Nit (optional): you can do this in one line:
```
const pageIdToConfig = {[pageElement.id]: config};
```
Note that the brackets are part of the [computed property name](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer#computed_property_names) syntax, which allows you to use any expression as a key as long as it is placed within brackets `[]`

Reply:
Yes, good to know, Corey! I'll take a note of this for future PRs.